examples/with-mobx/components/Page.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import Link from 'next/link'
+import { inject, observer } from 'mobx-react'
+import Clock from '../components/Clock'
```

Comment:
No here. You need to have it.
It's just for stateless components.
Keep it here.